site/sfguides/src/build-a-cinebot-with-snowflake-native-apps-and-cortex/build-a-cinebot-with-snowflake-native-apps-and-cortex.md

```diff
@@ -0,0 +1,255 @@
+author: Charles Yorek
+id: build-a-cinebot-with-snowflake-native-apps-and-cortex
+summary: This is a sample Snowflake Guide
+categories: Getting-Started
+environments: web
+status: Published 
+feedback link: https://github.com/Snowflake-Labs/sfguides/issues
+tags: Getting Started, Native Apps, Cortex, Data Applications
+
+# Build a CineBot with Snowflake Native Apps and Cortex
```

Comment:
@sfc-gh-cyorek Pl change the title to "Build a CineBot with Snowflake Native Apps and Snowfake Cortex"

Reply:
Updated